ports/foonathan-lexy/vcpkg.json

```
@@ -1,10 +1,10 @@
 {
   "name": "foonathan-lexy",
-  "version": "2022.12.1",
+  "version-string": "2025.05.0",
```

```suggestion
  "version": "2025.5.0",
```
because version-string isn't compatible with `version>=`.